Marlin/src/module/tool_change.cpp

```diff
@@ -884,8 +888,25 @@ void tool_change(const uint8_t tmp_extruder, const float fr_mm_s/*=0.0*/, bool n
           apply_motion_limits(destination);
         #endif
 
-        // Move back to the original (or tweaked) position
-        do_blocking_move_to(destination);
+        if (false
+          #if ENABLED(TOOLCHANGE_RETURN_NEXT_MOVE)
+            || true
+          #endif
+          #if ENABLED(DUAL_X_CARRIAGE)
+            || dual_x_carriage_mode == DXC_FULL_CONTROL_MODE
+          #endif
```

Comment:
…which means that this test will never evaluate to true.

Reply:
Which means the above is intended to operate in the opposite condition as this and theyre not nested.

Reply:
Not nested, right! I need to up my meds.